channeld/test/run-full_channel.c

```diff
@@ -120,7 +120,7 @@ static const struct htlc **include_htlcs(struct channel *channel, enum side side
 		struct preimage preimage;
 		struct sha256 hash;
 		enum channel_add_err e;
-		enum side sender;
+		enum side sender = 0;
```

Comment:
We should use the proper `enum side` label.

    /* Invalid value that will be written in the `switch` later -- only used to shut up the lint.  */
    enum side sender = NUM_SIDES;

The later `switch` statement properly initializes `sender`, this is just an egregious mistake by the compiler.